<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendFrame</class>
 <widget class="QFrame" name="SendFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="WalletGUI::ResizableScrollArea" name="m_sendScrollarea">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>654</width>
        <height>68</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="m_transfersLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="m_sendFeeFrame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <property name="horizontalSpacing">
       <number>18</number>
      </property>
      <item row="6" column="3">
       <widget class="QLineEdit" name="m_paymentIdEdit">
        <property name="styleSheet">
         <string notr="true">[errorState=&quot;true&quot;] {
  border-color: #ef3131;
}</string>
        </property>
        <property name="placeholderText">
         <string>Payment identifier issued by payee (market, exchange or other entity)</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QLabel" name="m_paymentIdTextLabel">
        <property name="text">
         <string>Payment ID</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="m_addRecipientButton">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
          <property name="text">
           <string>Add recipient</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>20</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="minimalLabel">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Slow</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="fastLabel">
           <property name="text">
            <string>Fast</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="0" colspan="3">
       <widget class="QSlider" name="m_feeSlider">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>4</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QLabel" name="m_feeTextLabel">
        <property name="toolTip">
         <string>Atomic units per byte</string>
        </property>
        <property name="text">
         <string>Fee (AU per byte)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="m_mixinFrame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>23</number>
      </property>
      <item row="0" column="2" rowspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="5" rowspan="2">
       <widget class="QLabel" name="m_totalAmountLabel">
        <property name="text">
         <string>amount</string>
        </property>
        <property name="indent">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="m_mixinSpin">
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3" rowspan="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>298</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QSlider" name="m_mixinSlider">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maximum">
         <number>50</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>6</number>
        </property>
        <property name="tracking">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="7" rowspan="2">
       <widget class="QPushButton" name="m_sendButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>218</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>218</width>
          <height>40</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" rowspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Total amount</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="m_mixinTextLabel">
        <property name="text">
         <string>Anonymity level</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6" rowspan="2">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WalletGUI::ResizableScrollArea</class>
   <extends>QScrollArea</extends>
   <header>resizablescrollarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_sendScrollarea</tabstop>
  <tabstop>m_addRecipientButton</tabstop>
  <tabstop>m_feeSlider</tabstop>
  <tabstop>m_paymentIdEdit</tabstop>
  <tabstop>m_mixinSpin</tabstop>
  <tabstop>m_mixinSlider</tabstop>
  <tabstop>m_sendButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_sendButton</sender>
   <signal>clicked()</signal>
   <receiver>SendFrame</receiver>
   <slot>sendClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>561</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_mixinSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SendFrame</receiver>
   <slot>mixinValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>520</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_addRecipientButton</sender>
   <signal>clicked()</signal>
   <receiver>SendFrame</receiver>
   <slot>addRecipientClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_paymentIdEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>SendFrame</receiver>
   <slot>validatePaymentId(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>538</x>
     <y>521</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_mixinSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SendFrame</receiver>
   <slot>mixinValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>575</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_feeSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SendFrame</receiver>
   <slot>feeValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sendClicked()</slot>
  <slot>clearAllClicked()</slot>
  <slot>addRecipientClicked()</slot>
  <slot>mixinValueChanged(int)</slot>
  <slot>validatePaymentId(QString)</slot>
  <slot>feeValueChanged(int)</slot>
 </slots>
</ui>
